atomate/vasp/workflows/base/library/neb.yaml

```diff
@@ -0,0 +1,53 @@
+# A standard CI-NEB calculation
```

Comment:
IDPP has been added to **WriteNEBFromEndpoints** (atomate.vasp.firetasks.neb_tasks) and set to default.